.tekton/rbac-permissions-operator-pull-request.yaml

```diff
@@ -8,10 +8,8 @@ metadata:
     build.appstudio.redhat.com/target_branch: '{{target_branch}}'
     pipelinesascode.tekton.dev/cancel-in-progress: 'true'
     pipelinesascode.tekton.dev/max-keep-runs: '3'
-    pipelinesascode.tekton.dev/on-cel-expression: "event == \"pull_request\"\n&& target_branch\
-      \ == \"master\"\n&& (\"{.,**}/*.go\".pathChanged() ||\n    \"{.,**}/go.*\".pathChanged()\
-      \ ||\n    \"build/Dockerfile\".pathChanged() ||\n    \".tekton/rbac-permissions-operator-pull-request.yaml\"\
-      .pathChanged() )\n"
+    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch
```

Comment:
This looks straight forward :D   @MitaliBhalla , could you help me understand why we have this change? or it is from a standard template?